libraries/src/Console/UpdateCoreCommand.php

```diff
@@ -386,4 +400,34 @@ public function copyFileTo($file, $dir): void
     {
         Folder::copy($file, $dir, '', true);
     }
+
+    /**
+     * Check Database Table Structure
+     *
+     * @return  integer the number of errors
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public function checkSchema(): int
+    {
+        $app       = $this->getApplication();
+        $app->getLanguage()->load('com_installer', JPATH_ADMINISTRATOR);
+        $errors            = 0;
+        $coreExtensionInfo = ExtensionHelper::getExtensionRecord('joomla', 'file');
+
+        $dbmodel = $app->bootComponent('com_installer')->getMVCFactory($app)->createModel('Database', 'Administrator');
+
+        // Ensure we only get information for core
+        $dbmodel->setState('filter.extension_id', $coreExtensionInfo->extension_id);
+
+        // We're filtering by a single extension which must always exist - so can safely access this through element 0 of the array
+        $changeInformation = $dbmodel->getItems()[0];
+        $errors            = $changeInformation['errorsCount'];
+
+        foreach ($changeInformation['errorsMessage'] as $msg) {
+            $this->ioStyle->info($msg);
+        }
+
+        return $errors;
```

Comment:
```suggestion
        $app = $this->getApplication();
        $app->getLanguage()->load('com_installer', JPATH_ADMINISTRATOR);
        $coreExtensionInfo = ExtensionHelper::getExtensionRecord('joomla', 'file');

        $dbmodel = $app->bootComponent('com_installer')->getMVCFactory($app)->createModel('Database', 'Administrator');

        // Ensure we only get information for core
        $dbmodel->setState('filter.extension_id', $coreExtensionInfo->extension_id);

        // We're filtering by a single extension which must always exist - so can safely access this through element 0 of the array
        $changeInformation = $dbmodel->getItems()[0];

        foreach ($changeInformation['errorsMessage'] as $msg) {
            $this->ioStyle->info($msg);
        }

        return $changeInformation['errorsCount'];
```
Small code style change and simplification by removing the `$errors` variable.

Reply:
thanks